Userland/DynamicLoader/main.cpp

```diff
@@ -149,7 +158,8 @@ NAKED void _start(int, char**, char**)
 
 ALWAYS_INLINE static void optimizer_fence()
 {
-    asm("" ::: "memory");
```

Comment:
Sometimes I hate `clang-format`...

Reply:
I'm still very confused which version of clang-format you use to get this formatting :P